### PR TITLE
Remove order url and price

### PR DESCRIPTION
**Pull request by JonathanHallam**
Trello - https://trello.com/c/MZ6js01b/1441-simplify-order-a-copy-implementation-in-whitehall

Removal of Price and Order URL from attachments

The order URL should be the same and the prices are often out of date. This PR removes the option for inputting a price or order URL and shows a link on official documents that lets users know how to purchase them. 

There will be a further PR to remove the fields and data from the database

### CODE REVIEW

File: config/locales/et.yml [hunk 1]
```diff
@@ -217,7 +217,6 @@ et:
         command_paper_number:
         hoc_paper_number: 'Parlamendi alamkoja dokumendi nr. '
         isbn: 'Manuse ISBN '
-        order_url: Manuse tellimise link
```
Comment (leenagupte) [inline config/locales/et.yml]: Is this something that's used in the frontend or in whitehall-admin? It looks like user facing content, so perhaps it should be kept in?

File: config/locales/en.yml [hunk 2]
```diff
@@ -68,7 +68,6 @@ en:
       command_paper_number: Attachment command paper number
       hoc_paper_number: House of Commons paper number
       isbn: Attachment ISBN
-      price: Attachment price
```
Comment (leenagupte) [inline config/locales/en.yml]: Here too. Is this something that's used in the frontend or whitehall-admin?

Comment (leenagupte) [reply config/locales/en.yml]: I think that by removing this, we are losing the internationalisation on the frontend.

File: config/locales/et.yml (hunk 1 above)
Comment (JonathanHallam) [reply config/locales/et.yml]: It is user facing but the idea is that it's not being used any more. There's now just a link to generic information that doesn't have a heading. Also from what I can tell the internationalization wasn't being used for order_url or price_in_pence so removing it doesn't really matter.

File: config/locales/en.yml (hunk 2 above)
Comment (JonathanHallam) [reply config/locales/en.yml]: I don't think so, I've removed all mention of order_url and I didn't take out any internationalisation, I think it's probably just in here to mirror the model.

Comment (JonathanHallam) [reply config/locales/en.yml]: actually scratch that, I've just spotted something that I missed I think I need to change something around a bit. I think it's safe to remove `order_url` and `price` in the way that I have because they're only used when inputting but I need to use `order_a_copy_full` when presenting the link.

Comment (leenagupte) [reply config/locales/en.yml]: Whilst we're here, can you move the I18n removals to separate commits? There's a lot of them, so you have to scroll a long way to see which tests have been removed.

File: app/views/documents/_attachment.html.erb [hunk 3]
```diff
@@ -56,6 +56,11 @@
         <%= attachment_attributes(attachment) %>
       <% end %>
     </p>
+    <% if attachment.is_official_document? %>
```
Comment (leenagupte) [inline app/views/documents/_attachment.html.erb]: Should this have a test?

File: app/models/attachment.rb [hunk 4]
```diff
@@ -54,6 +54,10 @@ def self.parliamentary_sessions
     end
   end
 
+  def is_official_document?
+    is_command_paper? || is_act_paper?
+  end
```
Comment (leenagupte) [inline app/models/attachment.rb]: Should this have a test?

Comment (JonathanHallam) [reply app/models/attachment.rb]: I don't think so, both `is_command_paper` and `is_act_paper` have tests and this is super simple logic. If you take a look at https://github.com/alphagov/whitehall/blob/master/test/unit/attachable_test.rb#L209 you can see a similar pattern; a `has_official_document` method exists that does the same thing but isn't tested because `has_command_paper` and `has_act_paper` are.

Happy to write one if you'd like though :)

File: app/views/documents/_attachment.html.erb (hunk 3 above)
Comment (JonathanHallam) [reply app/views/documents/_attachment.html.erb]: I'm happy that this is tested further down the line which is why i've not written one but will write one if you think it needs it :)

File: app/views/documents/_attachment.html.erb [hunk 5]
```diff
@@ -56,14 +56,9 @@
         <%= attachment_attributes(attachment) %>
       <% end %>
     </p>
-    <% if attachment.order_url.present? %>
-      <p>
-        <%= link_to t('attachment.headings.order_a_copy'), attachment.order_url,
+    <% if attachment.is_official_document? %>
+      <%= link_to t('attachment.headings.order_a_copy'), "https://www.gov.uk/guidance/how-to-buy-printed-copies-of-official-documents",
```
Comment (kevindew) [inline app/views/documents/_attachment.html.erb]: Sorry ignorant drive-by comment. Should this always be www.gov.uk or should we use Plek to have this be environment specific?

Comment (leenagupte) [reply app/views/documents/_attachment.html.erb]: Good spot! 🤦‍♂

Comment (JonathanHallam) [reply app/views/documents/_attachment.html.erb]: poop, yeah fair play, I'll raise a pr to fix this
